+ source ./common/common_lib.sh
++ source /root/mugen/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Sat Aug 26 16:57:37 2023 - INFO  - Start to prepare the test environment.
+ iSulad_install
+ DNF_INSTALL iSulad
+ pkgs=iSulad
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs iSulad --node 1 --tempfile ''
+ tmpfile2=/tmp/tmp0gxm_lfo
+ '[' -z '' ']'
+ tmpfile=/tmp/tmp0gxm_lfo
+ sed -i '/registry-mirrors/a\"docker.io"' /etc/isulad/daemon.json
+ systemctl restart isulad
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Sat Aug 26 16:57:45 2023 - INFO  - Start to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Sat Aug 26 16:57:45 2023 - INFO  - Start to run test.
++ hostname
oe_test_iSulad_resource_mapping.sh: line 30: hostname: command not found
+ isula run -itd -h --name volume -v vol:/vol:rw,nocopy centos
Invalid container hostname (--name), only ^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\-]*[A-Za-z0-9])$ and less than 64 bytes are allowed.
+ CHECK_RESULT 125 0 0 'create container failed'
+ actual_result=125
+ expect_result=0
+ mode=0
+ error_log='create container failed'
+ exit_mode=0
+ '[' -z 125 ']'
+ '[' 0 -eq 0 ']'
+ test 125x '!=' 0x
+ test -n 'create container failed'
+ LOG_ERROR 'create container failed'
+ message='create container failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'create container failed'
Sat Aug 26 16:57:45 2023 - ERROR - create container failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iSulad_resource_mapping.sh line 31'
+ message='oe_test_iSulad_resource_mapping.sh line 31'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iSulad_resource_mapping.sh line 31'
Sat Aug 26 16:57:45 2023 - ERROR - oe_test_iSulad_resource_mapping.sh line 31
+ '[' 0 -eq 1 ']'
+ return 0
+ expect
++ hostname
oe_test_iSulad_resource_mapping.sh: line 32: hostname: command not found
++ hostname
oe_test_iSulad_resource_mapping.sh: line 32: hostname: command not found
++ hostname
oe_test_iSulad_resource_mapping.sh: line 32: hostname: command not found
spawn isula exec -it volume
Error response from daemon: No such image or container or accelerator:volume
expect: spawn id exp3 not open
    while executing
"expect {
            "root@" {
                send "echo 'hello world' > test\r"
            }
        }"
+ grep 'hello world' /var/lib/isulad/volumes/vol/_data/test
grep: /var/lib/isulad/volumes/vol/_data/test: No such file or directory
+ CHECK_RESULT 2 0 0 'resource mapping failed'
+ actual_result=2
+ expect_result=0
+ mode=0
+ error_log='resource mapping failed'
+ exit_mode=0
+ '[' -z 2 ']'
+ '[' 0 -eq 0 ']'
+ test 2x '!=' 0x
+ test -n 'resource mapping failed'
+ LOG_ERROR 'resource mapping failed'
+ message='resource mapping failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'resource mapping failed'
Sat Aug 26 16:57:45 2023 - ERROR - resource mapping failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iSulad_resource_mapping.sh line 52'
+ message='oe_test_iSulad_resource_mapping.sh line 52'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iSulad_resource_mapping.sh line 52'
Sat Aug 26 16:57:45 2023 - ERROR - oe_test_iSulad_resource_mapping.sh line 52
+ '[' 0 -eq 1 ']'
+ return 0
++ hostname
oe_test_iSulad_resource_mapping.sh: line 53: hostname: command not found
+ isula run -itd -h --name volume1 --mount type=volume,src=vol1,dst=/vol1,volume-nocopy=true centos
Invalid container hostname (--name), only ^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\-]*[A-Za-z0-9])$ and less than 64 bytes are allowed.
+ CHECK_RESULT 125 0 0 'create container failed'
+ actual_result=125
+ expect_result=0
+ mode=0
+ error_log='create container failed'
+ exit_mode=0
+ '[' -z 125 ']'
+ '[' 0 -eq 0 ']'
+ test 125x '!=' 0x
+ test -n 'create container failed'
+ LOG_ERROR 'create container failed'
+ message='create container failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'create container failed'
Sat Aug 26 16:57:45 2023 - ERROR - create container failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iSulad_resource_mapping.sh line 54'
+ message='oe_test_iSulad_resource_mapping.sh line 54'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iSulad_resource_mapping.sh line 54'
Sat Aug 26 16:57:45 2023 - ERROR - oe_test_iSulad_resource_mapping.sh line 54
+ '[' 0 -eq 1 ']'
+ return 0
+ expect
++ hostname
oe_test_iSulad_resource_mapping.sh: line 55: hostname: command not found
++ hostname
oe_test_iSulad_resource_mapping.sh: line 55: hostname: command not found
++ hostname
oe_test_iSulad_resource_mapping.sh: line 55: hostname: command not found
spawn isula exec -it volume1
Error response from daemon: No such image or container or accelerator:volume1
expect: spawn id exp3 not open
    while executing
"expect {
            "root@" {
                send "echo 'hello world' > test\r"
            }
        }"
+ grep 'hello world' /var/lib/isulad/volumes/vol1/_data/test
grep: /var/lib/isulad/volumes/vol1/_data/test: No such file or directory
+ CHECK_RESULT 2 0 0 'resource mapping failed'
+ actual_result=2
+ expect_result=0
+ mode=0
+ error_log='resource mapping failed'
+ exit_mode=0
+ '[' -z 2 ']'
+ '[' 0 -eq 0 ']'
+ test 2x '!=' 0x
+ test -n 'resource mapping failed'
+ LOG_ERROR 'resource mapping failed'
+ message='resource mapping failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'resource mapping failed'
Sat Aug 26 16:57:45 2023 - ERROR - resource mapping failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iSulad_resource_mapping.sh line 75'
+ message='oe_test_iSulad_resource_mapping.sh line 75'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iSulad_resource_mapping.sh line 75'
Sat Aug 26 16:57:45 2023 - ERROR - oe_test_iSulad_resource_mapping.sh line 75
+ '[' 0 -eq 1 ']'
+ return 0
++ hostname
oe_test_iSulad_resource_mapping.sh: line 76: hostname: command not found
+ isula run -itd -h --name volume2 --volumes-from volume --volumes-from volume1 centos
Invalid container hostname (--name), only ^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\-]*[A-Za-z0-9])$ and less than 64 bytes are allowed.
+ CHECK_RESULT 125 0 0 'create container failed'
+ actual_result=125
+ expect_result=0
+ mode=0
+ error_log='create container failed'
+ exit_mode=0
+ '[' -z 125 ']'
+ '[' 0 -eq 0 ']'
+ test 125x '!=' 0x
+ test -n 'create container failed'
+ LOG_ERROR 'create container failed'
+ message='create container failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'create container failed'
Sat Aug 26 16:57:45 2023 - ERROR - create container failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iSulad_resource_mapping.sh line 77'
+ message='oe_test_iSulad_resource_mapping.sh line 77'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iSulad_resource_mapping.sh line 77'
Sat Aug 26 16:57:45 2023 - ERROR - oe_test_iSulad_resource_mapping.sh line 77
+ '[' 0 -eq 1 ']'
+ return 0
+ expect
++ hostname
oe_test_iSulad_resource_mapping.sh: line 78: hostname: command not found
++ hostname
oe_test_iSulad_resource_mapping.sh: line 78: hostname: command not found
++ hostname
oe_test_iSulad_resource_mapping.sh: line 78: hostname: command not found
spawn isula exec -it volume2
Error response from daemon: No such image or container or accelerator:volume2
expect: spawn id exp4 not open
    while executing
"expect {
            "root@" {
                send "grep 'hello world' /vol1/test\r"
            }
        }"
++ grep -c 'hello world' /tmp/log_volume
+ test 1 -eq 4
+ CHECK_RESULT 1 0 0 'resource mapping failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='resource mapping failed'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'resource mapping failed'
+ LOG_ERROR 'resource mapping failed'
+ message='resource mapping failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'resource mapping failed'
Sat Aug 26 16:57:45 2023 - ERROR - resource mapping failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iSulad_resource_mapping.sh line 99'
+ message='oe_test_iSulad_resource_mapping.sh line 99'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iSulad_resource_mapping.sh line 99'
Sat Aug 26 16:57:45 2023 - ERROR - oe_test_iSulad_resource_mapping.sh line 99
+ '[' 0 -eq 1 ']'
+ return 0
+ isula stop volume volume1 volume2
Error response from daemon: No such container:volume
No such container:volume
Error response from daemon: No such container:volume1
No such container:volume1
Error response from daemon: No such container:volume2
No such container:volume2
+ isula rm volume volume1 volume2
Error response from daemon: No such container:volume
Error response from daemon: No such container:volume1
Error response from daemon: No such container:volume2
+ grep 'hello world' /var/lib/isulad/volumes/vol/_data/test
grep: /var/lib/isulad/volumes/vol/_data/test: No such file or directory
+ CHECK_RESULT 2 0 0 'resource mapping failed'
+ actual_result=2
+ expect_result=0
+ mode=0
+ error_log='resource mapping failed'
+ exit_mode=0
+ '[' -z 2 ']'
+ '[' 0 -eq 0 ']'
+ test 2x '!=' 0x
+ test -n 'resource mapping failed'
+ LOG_ERROR 'resource mapping failed'
+ message='resource mapping failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'resource mapping failed'
Sat Aug 26 16:57:45 2023 - ERROR - resource mapping failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iSulad_resource_mapping.sh line 103'
+ message='oe_test_iSulad_resource_mapping.sh line 103'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iSulad_resource_mapping.sh line 103'
Sat Aug 26 16:57:45 2023 - ERROR - oe_test_iSulad_resource_mapping.sh line 103
+ '[' 0 -eq 1 ']'
+ return 0
+ grep 'hello world' /var/lib/isulad/volumes/vol1/_data/test
grep: /var/lib/isulad/volumes/vol1/_data/test: No such file or directory
+ CHECK_RESULT 2 0 0 'resource mapping failed'
+ actual_result=2
+ expect_result=0
+ mode=0
+ error_log='resource mapping failed'
+ exit_mode=0
+ '[' -z 2 ']'
+ '[' 0 -eq 0 ']'
+ test 2x '!=' 0x
+ test -n 'resource mapping failed'
+ LOG_ERROR 'resource mapping failed'
+ message='resource mapping failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'resource mapping failed'
Sat Aug 26 16:57:45 2023 - ERROR - resource mapping failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_iSulad_resource_mapping.sh line 105'
+ message='oe_test_iSulad_resource_mapping.sh line 105'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_iSulad_resource_mapping.sh line 105'
Sat Aug 26 16:57:45 2023 - ERROR - oe_test_iSulad_resource_mapping.sh line 105
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Sat Aug 26 16:57:45 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 8
+ test 8 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Sat Aug 26 16:57:45 2023 - ERROR - failed to execute the case.
+ exit 8
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Sat Aug 26 16:57:45 2023 - INFO  - Start to restore the test environment.
+ isula rmi centos
Error response from daemon: No such image:centos
+ iSulad_remove
+ systemctl stop isulad
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmp0gxm_lfo ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmp0gxm_lfo
+ '[' 0 -ne 0 ']'
+ rm -rf /etc/isulad
+ rm -rf /tmp/log_volume
+ LOG_INFO 'End to restore the test environment'
+ message='End to restore the test environment'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment'
Sat Aug 26 16:57:46 2023 - INFO  - End to restore the test environment
